app/src/main/res/values-it/strings.xml

```diff
@@ -33,6 +33,7 @@
     <!--Install-->
     <string name="keep_force_encryption">Mantieni crittografia forzata</string>
     <string name="keep_dm_verity">Mantieni AVB 2.0/dm-verity</string>
+    <string name="patch_vbmeta">Correggi vbmeta nell'immagine di boot</string>
```

Comment:
```suggestion
    <string name="patch_vbmeta">Correggi vbmeta nell\'immagine di boot</string>
```
Apostrophes need to be escaped with `\`, this fixes the automatic building as well

Reply:
D'Oh! I forgot it. Thanks.